app.rb

```diff
@@ -8,7 +8,8 @@ def list_all_book
   Book.book_list.each do |book|
     puts "Title: #{book.title}, Author: #{book.author}"
   end
-  main
+  Main.new.prompter
+  Director.new.director
```

Comment:
- [ ] As you can notice, this code has been repeated so many time in different method and that is against the DRY principle. Kindly let find a way to only run this in the `main.rb` file